packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import BlockAlignmentMatrixControl from '../';
+
+const meta = {
+	title: 'BlockEditor/BlockAlignmentMatrixControl',
+	component: BlockAlignmentMatrixControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'Renders a control for selecting block alignment using a matrix of alignment options.',
+			},
+		},
+	},
+	argTypes: {
+		label: {
+			control: 'text',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'Change matrix alignment' },
```

Comment:
```suggestion
				defaultValue: { summary: "'Change matrix alignment'" },
```

Reply:
While committing the code, the double quote (") automatically changes to single quote (')

Reply:
That's strange. I can commit normally in my environment:

https://github.com/user-attachments/assets/ffcfcb5d-d72e-4c26-b58c-9e23aedf7616

In your environment, what operation changes it to single quotes?

Reply:
Sorry for the trouble. It was my editor's fault, not a pre-commit hook as I thought. I’ve updated the PR.